ajax/libs/Hoverizr/package.json

```diff
@@ -25,5 +25,6 @@
   "repository": {
     "type": "git",
     "url": "git://github.com/iliasiovis/Hoverizr"
-  }
+  },
+  "license": "CC-BY"
```

Comment:
@LeaYeh we couldn't recognize "CC-BY" from our [license-list](https://github.com/cdnjs/cdnjs/blob/master/tools/license-list.json), could you ask the author which version of CC license he use? Thank you!